### PR TITLE
Refactor/user management

### PR DESCRIPTION
**Pull request by mimse**
This pull request includes significant changes to the user management feature in the admin module, primarily focusing on renaming components and updating their imports and templates. The changes aim to streamline the component structure and improve code readability.

### Component Renaming and Updates:

* [`libs/dh/admin/feature-user-management/src/index.ts`](diffhunk://#diff-6a96f187b3a732752a398dab1c51c5560fa9d4d0d333f9dd2c17d182258ea646L17-R17): Renamed `DhUsersOverviewComponent` to `DhUserManagementComponent` for better clarity.

### Template and Import Adjustments:

* [`libs/dh/admin/feature-user-management/src/lib/details/details.component.html`](diffhunk://#diff-147d09e0235e3025a53d3ff06a8c15195c47040586ba8f4be260abe20bfd66a8L68-R81): Updated the template to use `watt-tabs` and `dh-user-*` components, improving the organization of user details. [[1]](diffhunk://#diff-147d09e0235e3025a53d3ff06a8c15195c47040586ba8f4be260abe20bfd66a8L68-R81) [[2]](diffhunk://#diff-147d09e0235e3025a53d3ff06a8c15195c47040586ba8f4be260abe20bfd66a8L87-R98) [[3]](diffhunk://#diff-147d09e0235e3025a53d3ff06a8c15195c47040586ba8f4be260abe20bfd66a8L120-R131)
* [`libs/dh/admin/feature-user-management/src/lib/details/details.component.ts`](diffhunk://#diff-f7cd3fd7e0718593f2679f793ec6894ee565babb8b1f0cf81ab2305a8edfb67eL34): Removed unnecessary imports and updated component references to match the new naming conventions. [[1]](diffhunk://#diff-f7cd3fd7e0718593f2679f793ec6894ee565babb8b1f0cf81ab2305a8edfb67eL34) [[2]](diffhunk://#diff-f7cd3fd7e0718593f2679f793ec6894ee565babb8b1f0cf81ab2305a8edfb67eL52-R76)
* [`libs/dh/admin/feature-user-management/src/lib/details/tabs/audit-logs.component.ts`](diffhunk://#diff-99e48ecd7e26163ab3937f1ca3d11107ff07323e4e953f05bb14d1ff7d916e51L22): Renamed from `dh-user-audit-logs.component.ts` and updated imports. [[1]](diffhunk://#diff-99e48ecd7e26163ab3937f1ca3d11107ff07323e4e953f05bb14d1ff7d916e51L22) [[2]](diffhunk://#diff-99e48ecd7e26163ab3937f1ca3d11107ff07323e4e953f05bb14d1ff7d916e51L42-R41) [[3]](diffhunk://#diff-99e48ecd7e26163ab3937f1ca3d11107ff07323e4e953f05bb14d1ff7d916e51L60)
* [`libs/dh/admin/feature-user-management/src/lib/details/tabs/master-data.component.ts`](diffhunk://#diff-967f97688c6386bcb5dcd06ced816534657380cf0aedd91badfd081132e5d06cL32-R32): Renamed from `dh-user-master-data.component.ts` and updated imports.

### Removal of Deprecated Components:

* [`libs/dh/admin/feature-user-management/src/lib/drawer/tabs/dh-drawer-tabs.component.html`](diffhunk://#diff-39fc03beb8522a3e84477143197b0dfd7394f09a5b6045c8a7cbcdadbd0bc9c0L1-L29): Removed the old drawer tabs template.
* [`libs/dh/admin/feature-user-management/src/lib/drawer/tabs/dh-drawer-tabs.component.ts`](diffhunk://#diff-dd4d5487aea71d117071afc8fa81248ce3dfaa0d956a78e2d2b1df1b791db185L1-L49): Removed the old drawer tabs component.

### Other Changes:

* Updated various other component files to align with the new naming conventions and removed unnecessary imports. [[1]](diffhunk://#diff-ebee5649415885016848e5b4c2bef68b4fa8cdf04a8999988fd89d1dcb430d53L36) [[2]](diffhunk://#diff-ebee5649415885016848e5b4c2bef68b4fa8cdf04a8999988fd89d1dcb430d53L54-R58) [[3]](diffhunk://#diff-0a5b9487c7d00082bdb160feef4d3908397a028787bef19cc8a658ee3bd4c556L21-L30) [[4]](diffhunk://#diff-0a5b9487c7d00082bdb160feef4d3908397a028787bef19cc8a658ee3bd4c556L46-L47) [[5]](diffhunk://#diff-0a5b9487c7d00082bdb160feef4d3908397a028787bef19cc8a658ee3bd4c556L56-R53) [[6]](diffhunk://#diff-88a4245ff7b39730a80b64e31e748e427536299e8b5e18edcf7b2d031661b4b5L38) [[7]](diffhunk://#diff-88a4245ff7b39730a80b64e31e748e427536299e8b5e18edcf7b2d031661b4b5L63-R69) [[8]](diffhunk://#diff-88a4245ff7b39730a80b64e31e748e427536299e8b5e18edcf7b2d031661b4b5L79) [[9]](diffhunk://#diff-b333dda11f06470e860c5bdb1643e24263e5cc5c8b40f2464848be96fe9d776bL1-L73)

### CODE REVIEW

File: libs/dh/admin/feature-user-management/src/lib/filters/dh-filters.component.ts
Comment (dzhavat) [inline libs/dh/admin/feature-user-management/src/lib/filters/dh-filters.component.ts:1]: nitpick: drop `dh-` prefix from file name.

File: libs/dh/admin/feature-user-management/src/lib/dh-user-latest-login.component.ts
Comment (dzhavat) [inline libs/dh/admin/feature-user-management/src/lib/dh-user-latest-login.component.ts:1]: nitpick: drop `dh-` prefix from file name.

File: libs/dh/admin/feature-user-management/src/lib/filters/dh-filters.component.ts
Comment (mimse) [reply libs/dh/admin/feature-user-management/src/lib/filters/dh-filters.component.ts:1]: Fixed

File: libs/dh/admin/feature-user-management/src/lib/dh-user-latest-login.component.ts
Comment (mimse) [reply libs/dh/admin/feature-user-management/src/lib/dh-user-latest-login.component.ts:1]: Fixed

**Review by dzhavat — APPROVED** [review]
praise:

**Pull request merged**